feed_aggregated_test.go

```diff
@@ -6,16 +6,18 @@ import (
 	"time"
 
 	getstream "github.com/GetStream/stream-go"
+	"github.com/stretchr/testify/assert"
 )
 
-func getAggregatedFeed(client *getstream.Client) *getstream.AggregatedFeed {
-	f, _ := client.AggregatedFeed("aggregated", RandString(8))
+func getAggregatedFeed(t *testing.T, client *getstream.Client) *getstream.AggregatedFeed {
+	f, err := client.AggregatedFeed("aggregated", RandString(8))
+	assert.Nil(t, err)
```

Comment:
You should use `require.NoError`, it prints the error better and it quits the test instantly. This should be done in other places as well.